app/editor/menus/block.tsx

```diff
@@ -205,5 +205,11 @@ export default function blockMenuItems(dictionary: Dictionary): MenuItem[] {
       keywords: "diagram flowchart",
       attrs: { language: "mermaidjs" },
     },
+    {
+      name: "code_block",
+      title: "Kroki Diagrams",
+      keywords: "diagram flowchart",
+      attrs: { language: "kroki", diagram: "c4plantuml" },
+    },
```

Comment:
I think the [official favicon](https://github.com/yuzutech/kroki.io/blob/master/assets/favicon-128.png) would be a suitable icon for the block entry. However, it should be noted that while [the Kroki (repo) itself is under the MIT licence](https://github.com/yuzutech/kroki/blob/main/LICENSE), the "kroki.io" [website repository](https://github.com/yuzutech/kroki.io) does not have a licence file, so it might be necessary - or rather legally correct - to create an issue and ask for a usage permission.

Reply:
The [license](https://github.com/yuzutech/kroki.io/blob/master/LICENSE) for kroki.io is Mozilla Public License 2.0